README.md

```diff
@@ -1 +1,68 @@
-# decode-a-morse-code-message
\ No newline at end of file
+# decode-a-morse-code-message
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Decode a Morse code message
+
+> Basic logic to decode Morse code message.
+
+
+## Built With
+
+- Ruby
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
```

Comment:
- **[OPTIONAL]** I would recommend you include the steps on how to run the project locally.

Reply:
@alphayowakarindi Agreed, thanks!